src/server/api/memobase_server/models/database.py

```diff
@@ -409,6 +410,10 @@ class UserEvent(Base):
         foreign_keys=[user_id, project_id],
     )
 
+    embedding: Mapped[Vector] = mapped_column(
```

Comment:
Can we do something like `Vector Index` here https://github.com/pgvector/pgvector?tab=readme-ov-file#filtering.
Since we only need to index embedding in the same project

Reply:
never mind, seem like if we're using default vector index, pg-vector will use the DB index instead of a specific vector index.